docs/commands.rst

```diff
@@ -0,0 +1,36 @@
+.. _commands:
+
+Commands
+--------
+
+.. _cmd-changelog:
+
+changelog
+^^^^^^^^^
+When executed this command will generate/update ``CHANGELOG.txt``.

```

Comment:
This is not true, the changelog command prints the changelog to stdout. If the option `--post` is passed it will be posted on github using the `GH_TOKEN` similar to what publish does.